presentations/en/2019-11-05-CIOEC-Mandate.md

```diff
@@ -26,64 +29,65 @@ ESDC corporate risk profile identifies IT as the #1 risk to the organization
 
 ## IT Strategy team's mission
 
-<!-- markdownlint-disable -->
-*Reason for existence*
-
 Enable the strategic value of IT within ESDC by ***reducing its risks*** and ***increasing business agility***.
-<!-- markdownlint-enable -->
 
 ---
 
 ## Mandate (authority)
 
-<!-- markdownlint-disable -->
-1. Diagnose key problems
-  * organizational problems
-  * waste
-  * impediments to rapid lead times, and
-  * governance structures
-<!-- markdownlint-enable -->
+--
+
+### 1. Diagnose
+
+***Diagnose key problems:***
 
-The aforementioned items inhibit the desired changes in behaviours required
- for a digital-first government
+- organizational problems
+- waste
+- impediments to rapid lead times, and
+- governance structures
+
+The aforementioned items inhibit the desired changes in behaviours required for a digital-first government
 
 --
-<!-- markdownlint-disable -->
-2. Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks 
-<!-- markdownlint-enable -->
+
+### 2. Recommend
+
+Recommend ***specific course of actions*** to governance bodies to adopt new methods of managing IT risks
 
 --
 
-<!-- markdownlint-disable -->
-3. Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
-<!-- markdownlint-enable -->
+### 3. Measure
+
+***Measure progress of adoption*** of strategies in IT teams and governance bodies as a ***performance indicator*** of the IT Strategy team
```

Comment:
By our own definitions, shouldn't the strategies be measured against our performance in general (which is itself measured against business performance?)

Reply:
I'd be against that as it puts way too much dependencies on us.
Business performance would be measured against multiple different building blocks. We have an influence on those building blocks and we I say we can measure that influence instead of measuring those individual team's performance as a reflection of our IT Strategies